doc/man3/SSL_get_certificate.pod

```diff
@@ -0,0 +1,53 @@
+=pod
+
+=head1 NAME
+
+SSL_get_certificate, SSL_get_privatekey - retrieve TLS/SSL certificate and
+private key
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ X509 *SSL_get_certificate(const SSL *s);
+ EVP_PKEY *SSL_get_privatekey(const SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_get_certificate() returns a pointer to the B<X509> object representing the
+certificate which represents the local peer's identity. For servers, it returns
+the server's certificate. If a server has multiple certificates (for different
+algorithms, for example RSA and ECDSA), the certificate in actual use is
+returned. For clients, it returns the client certificate being used, if any.
+
+A specific use for SSL_get_certificate() is inside a callback set via a call to
+L<SSL_CTX_set_tlsext_status_cb(3)>, where it can be used to examine a server's
+certificate, for example for the purpose of identifying a certificate's OCSP
+responder URL so that an OCSP response can be obtained.
+
+SSL_get_privatekey() returns a pointer to the B<EVP_PKEY> object corresponding
+to the certificate returned by SSL_get_certificate().
+
+=head1 RETURN VALUES
+
+These functions return pointers to their respective objects, or NULL if no such
+object is available. Before the handshake has completed, the certificate
+returned is the one most recently added to the SSL object, or NULL if no
+certificate has yet been added. After the handshake has completed, the
+certificate returned is the one which was selected during the handshake.
```

Comment:
I'd keep this more about about error conditions etc, e.g. "SSL_get_certifciate() returns a pointer to the certificate or a NULL if there is no certificate"